source/codegen/metadata/nifgen/CHANGES.md

```diff
@@ -87,3 +87,26 @@ Following attributes removed as Calibrations is no longer supported for gRPC:
 - `OSCILLATOR_FREQ_DAC_VALUE`
 - `POST_AMPLIFIER_ATTENUATION`
 - `PRE_AMPLIFIER_ATTENUATION`
+
+## enums.py
+
+All occurrences of the following keys were removed along with their values:
+- `'documentation'`
+
+Metadata for following enums missing in enums.hapigen, taken from constants.hapigen:
+- `'FrequencyListHandleParameter'`
```

Comment:
Once you rename the enum, make sure to update the name here as well.

Reply:
I have updated the name in changes.md as well